### ISSUE TITLE
Error mlcli - when training regression test.

### ISSUE THREAD

**Issue by julianobd** [dotnet/machinelearning-samples#652]
`mlnet auto-train --task regression --dataset taxi-fare-train.csv --test-dataset taxi-fare-test.csv --label-column-name fare_amount --max-exploration-time 1000`

return:

`Exploring multiple ML algorithms and settings to find you the best model for ML task: regression
For further learning check: https://aka.ms/mlnet-cli
▓▓▓▓▓▓▓▓▓▓▓─────────────────────────────────────────────────────────────────────────────────────────────────────────────Best quality(RSquared): 0,9440, Best Algorithm: LightGbmRegression, Last Algorithm: FastTreeTweedieRegression   00:01:37Exception occured while exploring pipelines:
All instances skipped due to missing features.
Please see the log file for more info.`

Log:
[debug_log.txt](https://github.com/dotnet/machinelearning-samples/files/3611022/debug_log.txt)

Datasets:
[teste.zip](https://github.com/dotnet/machinelearning-samples/files/3611028/teste.zip)

**Comment by CESARDELATORRE** [comment]
Can you try with a single input dataset (so the CLI will internally use cross-validation instead of provided split files) just to isolate if the issue is with the test-dataset?

**Comment by julianobd** [comment]
I've just tryied using only train dataset and only test dataset.
[train_dataset_debug_log.txt](https://github.com/dotnet/machinelearning-samples/files/3611853/train_dataset_debug_log.txt)
[test_dataset_debug_log.txt](https://github.com/dotnet/machinelearning-samples/files/3611859/test_dataset_debug_log.txt)

commands used:
`mlnet auto-train --task regression --dataset taxi-fare-test.csv --label-column-name fare_amount --max-exploration-time 1000`

`mlnet auto-train --task regression --dataset taxi-fare-test.csv --label-column-name fare_amount --max-exploration-time 1000`
